app/controllers/admin/articles_controller.rb

```diff
@@ -26,6 +28,7 @@ def new
   # /admin/articles/1/edit
   def edit
     @collection = Article.find(@article.collection_id) if @article.in_collection?
+    @title = t('.edit_title', id: @article.id, title: @article.title, subtitle: @article.subtitle)
```

Comment:
I wanted to put a method in `admin_controller` that looked something like this:

```ruby
def admin_title(action:, model:)
  t(".#{action}_title", id: model.id, title: model.title, subtitle: model.subtitle)
end
```

and then, for example, in `admin/articles_controller` just do 
```ruby
@title = admin_title(current_action, @article)
```

but not every model uses `title` (some use `name`) so it seemed like it was more confusing than just putting this `t(...)` line wherever it is needed